src/main/java/pages/WebViewPage.java

```diff
@@ -0,0 +1,49 @@
+package pages;
+
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.android.AndroidElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+
+import java.util.List;
+
+public class WebViewPage extends BasePage {
+
+    public WebViewPage() {
+        super();
+    }
+
+    @AndroidFindBy(xpath = "//android.widget.ListView/android.view.View/android.view.View")
+    private List<AndroidElement> someTextViews;
```

Comment:
Remember to leave a blank line between every property to make this more readable

Reply:
a blank space is left between each property